app/controllers/class_categories_controller.rb

```diff
@@ -0,0 +1,66 @@
+class ClassCategoriesController < ApplicationController
+  before_action :find_class_category, only: %i[edit update show destroy]
+  before_action :authenticate_user!
+  def index
+    @class_categories = ClassCategory.all
+  end
+
+  def edit
+    all_teachers
+  end
+
+  def update
+    if @class_category.update(class_category_params)
+      flash[:notice] = 'Categoria de Aula editada com sucesso.'
+      redirect_to class_category_path(class_category_params)
```

Comment:
```suggestion
      redirect_to class_category_path(class_category_params), notice: 'Categoria de Aula editada com sucesso.'
```

Daria pra fazer os dois na mesma linha e se colocar no I18n ainda ficaria top